frontend/app/components/layouts/public-layout.tsx

```diff
@@ -46,7 +46,7 @@ export function PublicLayout({ children }: PropsWithChildren) {
 export function AppPageTitle({ children }: PropsWithChildren) {
   const { t } = useTranslation(i18nNamespaces);
   return (
-    <div className="my-8 border-b border-red-800">
+    <div className="my-8 max-w-prose border-b border-red-800">
```

Comment:
Wrap the `h2` and `PageTitle` with another `div` and apply `max-w-prose` class to it. The heading bottom border should be full width.